Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/configuration/server/CCLConfigurationApiV1.kt

```diff
@@ -1,5 +1,11 @@
 package de.rki.coronawarnapp.ccl.configuration.server
 
+import okhttp3.ResponseBody
+import retrofit2.Response
+import retrofit2.http.GET
+
 interface CCLConfigurationApiV1 {
-    // DO
+
+    @GET("/version/v1/ccl/config")
```

Comment:
There was a Tech Spec Change https://github.com/corona-warn-app/cwa-app-tech-spec/pull/149/commits/56251878373b2bad8f3a03aa362ba2d902fbe288
The url needs to be changed to `version/v1/ccl/config-v1`